packages/airnode-feed/src/validation/schema.ts

```diff
@@ -167,33 +167,53 @@ const validateTriggerReferences: SuperRefinement<{
 }> = async (config, ctx) => {
   const { ois: oises, templates, endpoints, triggers } = config;
 
-  for (const signedApiUpdate of triggers.signedApiUpdates) {
+  for (const [signedApiUpdateIndex, signedApiUpdate] of triggers.signedApiUpdates.entries()) {
     const { templateIds } = signedApiUpdate;
 
+    // Verify all template IDs actually exist in the templates object
+    const referenceErrors = templateIds.map((templateId, templateIdIndex) => {
+      if (templates[templateId] === undefined) {
+        ctx.addIssue({
+          code: z.ZodIssueCode.custom,
+          message: `Template ID "${templateId}" is not defined in the config.templates object`,
```

Comment:
nit:
```suggestion
          message: `Template ID "${templateId}" is not defined`,
```

if we ever refactor the config it can go stale.

Reply:
> if we ever refactor the config it can go stale.

You're right but current version clearly points where the issue arises. I'd recommend to keep it.